mkdocs.yml

```diff
@@ -61,6 +61,7 @@ nav:
           - Overview: 'features/software-templates/index.md'
           - Configuration: 'features/software-templates/configuration.md'
           - Adding your own Templates: 'features/software-templates/adding-templates.md'
+          - Input Examples: 'features/software-templates/input-examples.md'
```

Comment:
It needs to be added to https://github.com/backstage/backstage/blob/master/microsite/sidebars.json as well, for it to show up on backstage.io